app/delivery/meteor_routes.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import getMeteors from "../useCases/fetch_meteors.js";
+import Exception from "../utils/exception.js";
+
+const meteorRouter = express.Router();
+
+meteorRouter.get('/meteors', async (req, res, next) => {
+    try {
+        const meteors = await getMeteors();
+
+        res.json(meteors);
+    } catch (error) {
+        const status = error.status || 500;
```

Comment:
you already have this line here https://github.com/SpanFish11/node-proxy-server/pull/7/files#diff-f35672458e83e5dbbb2cdd7d523de438dc10d6a30272994170e6558dd7c8860fR4